doc/source/sparse.rst

```diff
@@ -135,3 +135,84 @@ recommend using ``block`` as it's more memory efficient. The ``integer`` format
 keeps an arrays of all of the locations where the data are not equal to the
 fill value. The ``block`` format tracks only the locations and sizes of blocks
 of data.
+
+Interaction with scipy.sparse
+-----------------------------
+
+Experimental api to transform between sparse pandas and scipy.sparse structures.
+
+:meth:`SparseSeries.to_coo`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A ``SparseSeries.to_coo`` method is implemented for transforming a ``SparseSeries`` indexed by a ``MultiIndex`` to a ``scipy.sparse.coo_matrix``.
+
+The method requires a ``MultiIndex`` with two or more levels.
+
+.. ipython:: python
+   :suppress:
+
+   from numpy import nan
```

Comment:
kill the pd import and just use ``Series`` for example. BUT, do add the ``from numpy import nan`` line in the code (as you are directly using it, which is fine)